rust/Cargo.toml

```diff
@@ -49,3 +50,6 @@ features = [
 
 [profile.release]
 lto = true
+
+[patch.crates-io]
+audiopus_sys = { git = "https://github.com/ProjectAnni/audiopus_sys" }
```

Comment:
Use this fork as discussed earlier: https://github.com/erikas-taroza/audiopus_sys